src/object/obj_rpc.c

```diff
@@ -392,8 +439,3400 @@ crt_proc_struct_daos_shard_tgt(crt_proc_t proc, struct daos_shard_tgt *st)
 	return 0;
 }
 
-CRT_RPC_DEFINE(obj_update, DAOS_ISEQ_OBJ_RW, DAOS_OSEQ_OBJ_RW)
-CRT_RPC_DEFINE(obj_fetch, DAOS_ISEQ_OBJ_RW, DAOS_OSEQ_OBJ_RW)
+// CRT_RPC_DEFINE(obj_update, DAOS_ISEQ_OBJ_RW, DAOS_OSEQ_OBJ_RW)
+static int crt_proc_struct_obj_update_in(crt_proc_t proc,
+					 struct obj_update_in *ptr)
+{
+	int rc = 0;
+	rc = crt_proc_struct_dtx_id(proc, &ptr->orw_dti);
+	if (rc)
+		do {
+			__typeof__(rc) __rc = (rc);
+			(void) (__rc);
+			goto out;
+		}
+		while (0);
+	rc = crt_proc_daos_unit_oid_t(proc, &ptr->orw_oid);
+	if (rc)
+		do {
+			__typeof__(rc) __rc = (rc);
+			(void) (__rc);
+			goto out;
+		}
+		while (0);
+	rc = crt_proc_uuid_t(proc, &ptr->orw_pool_uuid);
+	if (rc)
+		do {
+			__typeof__(rc) __rc = (rc);
+			(void) (__rc);
+			goto out;
+		}
+		while (0);
+	rc = crt_proc_uuid_t(proc, &ptr->orw_co_hdl);
+	if (rc)
+		do {
+			__typeof__(rc) __rc = (rc);
+			(void) (__rc);
+			goto out;
+		}
+		while (0);
+	rc = crt_proc_uuid_t(proc, &ptr->orw_co_uuid);
+	if (rc)
+		do {
+			__typeof__(rc) __rc = (rc);
+			(void) (__rc);
+			goto out;
+		}
+		while (0);
+	rc = crt_proc_uint64_t(proc, &ptr->orw_epoch);
+	if (rc)
+		do {
+			__typeof__(rc) __rc = (rc);
+			(void) (__rc);
+			goto out;
+		}
+		while (0);
+	rc = crt_proc_uint64_t(proc, &ptr->orw_dkey_hash);
+	if (rc)
+		do {
+			__typeof__(rc) __rc = (rc);
+			(void) (__rc);
+			goto out;
+		}
+		while (0);
+	rc = crt_proc_uint32_t(proc, &ptr->orw_map_ver);
+	if (rc)
+		do {
+			__typeof__(rc) __rc = (rc);
+			(void) (__rc);
+			goto out;
+		}
+		while (0);
+	rc = crt_proc_uint32_t(proc, &ptr->orw_nr);
+	if (rc)
+		do {
+			__typeof__(rc) __rc = (rc);
+			(void) (__rc);
+			goto out;
+		}
+		while (0);
+	rc = crt_proc_uint32_t(proc, &ptr->orw_start_shard);
+	if (rc)
+		do {
+			__typeof__(rc) __rc = (rc);
+			(void) (__rc);
+			goto out;
+		}
+		while (0);
+	rc                  = crt_proc_uint32_t(proc, &ptr->orw_flags);
+	if (rc)
+		do {
+			__typeof__(rc) __rc = (rc);
+			(void) (__rc);
+			goto out;
+		}
+		while (0);
+	daos_key_t *tmp_iov = &ptr->orw_dkey;
+	if (tmp_iov->iov_buf_len < tmp_iov->iov_len) {
+		D_ERROR("RYON: Issue with IOV: iov buf len ("DF_U64")"
+								   " < iov len "DF_U64"\n",
+			tmp_iov->iov_buf_len, tmp_iov->iov_len);
+		print_trace();
+	}
+	rc = crt_proc_d_iov_t(proc, &ptr->orw_dkey);
+	if (rc)
+		do {
+			__typeof__(rc) __rc = (rc);
+			(void) (__rc);
+			goto out;
+		}
+		while (0);
```

Comment:
(style)  that open brace { should be on the previous line
(style)  while should follow close brace '}'